root/static/scripts/common/components/Annotation.js

```diff
@@ -92,14 +94,18 @@ const Annotation = ({
                 date: formatUserDate($c, annotation.creation_date),
                 user: <EditorLink editor={annotation.editor} />,
               })}
+              {' '}
               {numberOfRevisions && numberOfRevisions > 1 ? (
                 <>
-                  {' '}
                   <a href={entityHref(entity, '/annotations')}>
                     {l('View annotation history')}
                   </a>
+                  {' | '}
                 </>
               ) : null}
+              <a href={returnUri($c, entityHref(entity, 'edit_annotation'))}>
```

Comment:
Sorry for the confusion, but with the previous review comment I was more wondering if using `returnUri` is necessary at all here. I think just specifying  `href={entityHref(entity, 'edit_annotation')}` here is enough, unless I'm missing something.

Reply:
You were right, it's unneeded. Got rid!